tests/cache/models/samples/upsertBulk.js

```diff
@@ -342,7 +343,7 @@ describe('api::redisEnabled::POST::bulkUpsert ' + path, () => {
           expect(res.body[1].value).to.equal('10');
           done();
         });
-      }, 500);
+      }, 100);
```

Comment:
I love shortening the time it takes to run tests, but would this make it more flappy?